docs/src/docs/cli/installation.md

```diff
@@ -100,7 +100,7 @@ That's it!
 
 Make sure you get the correct download for your hardware.
 
-1. <a href="https://github.com/jozu-ai/kitops/releases/latest/download/kitops-linux-arm64.tar.gz"
+1. Linux: <a href="https://github.com/jozu-ai/kitops/releases/latest/download/kitops-linux-arm64.tar.gz"
```

Comment:
Similar to windows, 90%+ of linux users are going to be using x86_64.

Reply:
fixed